ignite/handlers/state_param_scheduler.py

```diff
@@ -329,21 +329,42 @@ class StepStateScheduler(StateParamScheduler):
 
     Examples:
 
-        .. code-block:: python
+        .. testsetup::
+
+            engine = get_default_trainer()
+
+        .. testcode::
 
-            ...
-            engine = Engine(train_step)
 
             param_scheduler = StepStateScheduler(
-                param_name="param", initial_value=10, gamma=0.99, step_size=5
-            )
+                param_name="param", initial_value=1, gamma=0.9, step_size=5, create_new=True
+                )
+
+            # parameter is param, initial_value sets param to 1, gamma is set as 0.9
+            # Epoch 1 to 4, param does not change as step size is 5,
+            # Epoch 5, param changes from 1 to 1*0.9, param = 0.9
+            # Epoch 5 to 9, param = 0.9 as step size is 5,
+            # Epoch 10, param changes from 0.9 to 0.9*0.9, param = 0.81
+            # Epoch 10 to 14, param = 0.81, as step size is 5
+            # Epoch 15, param changes from 0.81 to 0.81*0.9, param = 0.729
+            # and so on ... the param change at Epoch = 5, 10, 15, 20, . . .
 
             param_scheduler.attach(engine, Events.EPOCH_COMPLETED)
 
-            # basic handler to print scheduled state parameter
-            engine.add_event_handler(Events.EPOCH_COMPLETED, lambda _ : print(engine.state.param))
+            @engine.on(Events.EPOCH_COMPLETED)
```

Comment:
See the doc https://pytorch-ignite.ai/concepts/02-events-and-handlers/

We can use event filter:
```python
@engine.on(Events.EPOCH_COMPLETED(every=5))
def print_param():
    print(engine.state.param)
```

Reply:
@bibhabasumohapatra Could you address this ?

Reply:
If i do this... Then  it will 25 lines!... And  only  wanted to print changes!

Reply:
cool!  knew (every=5) but did not thought about it here !!  thanks again.